### PR TITLE
add parameter, hostname of container

### PR DESCRIPTION
**Pull request by kixiro**
Sometimes a hostname is required other than "localhost"

### CODE REVIEW

**Comment by vania-pooh** [comment]
@kixiro please explain what is the use case for this feature?

**Comment by aandryashin** [comment]
As I can see all created containers will have the same hostname.

2017-09-18 16:27 GMT+03:00 Ivan Krutov <notifications@github.com>:

**Comment by vania-pooh** [comment]
@kixiro we are asking because we doubt whether setting the same hostname for containers running in parallel is a good idea. E.g. if hostname = `my-hostname-prefix-<uuid>` - that's ok. When all containers have hostname = `example.com` this can lead to DNS issues.

**Comment by kixiro** [comment]
@vania-pooh. The application under test looks at the hostname of the system through the java applet. If it does not match the hostname of the application's configuration file, it does not work. I need to specify a fixed name other than localhost. my-hostname-prefix-<uuid> -it doesn't suit me.

File: session/session.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ type Caps struct {
 	VNC              bool   `json:"enableVNC"`
 	TestName         string `json:"name"`
 	TimeZone         string `json:"timeZone"`
+	ContainerHostname	string `json:"containerHostname"`
```
Comment (vania-pooh) [inline session/session.go]: You forgot to run `go fmt`. Formatting here seems to be broken.

**Comment by vania-pooh** [comment]
Thank you for contribution!

**Pull request merged**